### DISCUSSION TITLE
Odd timing behaviour on setup manager...

### DISCUSSION THREAD

**Discussion by AitchySan** [jamf/Setup-Manager#20] [General]
I had previously installed setup-manager on our dev. i was impressed as it bypassed a separate issue we have with our regular prestaging related to our eduroam/global wifi. it was working before, since then i have updated to the latest pkg and im seeing the following behaviour:

-machine is enrolled and installs the profs
- machine gets to the create local user account screen
- within about 5-10 seconds setup-manager takes focus and runs on top of the create account screen not allowing me to enter details
- setup-manager finishes successfully
- the device goes back to the create account screen which was presumably in the background
- no accounts can be created at this stage and the machine is botched...

anyone else seeing this? it wasnt doing this before and i havent changed much in the way of config profs etc. i will revert it to an older version to see if it does this still, but wondering if anyone has saw this issue at all? many thanks!

**Comment by scriptingosx** [comment]
Turn off the machine by holding the power button until it turns off and reboot, does that allow you to continue?

Are you installing something other than Setup Manager and its profile during the prestage? Try moving that out of the Prestage and install it with Setup Manager instead?

Are you running policies with the `enrollmentComplete` trigger? Try removing those from the `enrollmentComplete` trigger and running them with Setup Manager instead.


You can (and should) always look at the logs of Jamf (at `/var/log/jamf.log`) and Setup Manager (at `/Library/Logs/Setup Manager.log`) and compare what is going in both for troubleshooting.

**Reply by AitchySan** [reply]
i will try this tomorrow as im at home just now, but the setup is pretty bare bones! i have:
 setup-manager pkg
jamf connect pkg
jamf connect launch agent
some branding files in a pkg

config profs:

jamf connect auth changer
setup manager prof 
filevault prof

the setup manager prof is pretty basic too, just firing out about 6 very common apps. 

i havent looked at the logs as i just finished running it yesterday late, but il will do this tomorrow morning first thing to see if it reveals anything. should have checked that first to be fair, i was just confused as it was working before. very much appreciate your feedback!!

**Reply by scriptingosx** [reply]
> Are you installing something other than Setup Manager and its profile during the prestage? Try moving that out of the Prestage and install it with Setup Manager instead?

This is key here. If JamfConnect.pkg does some... funky... stuff at installation that will mess with Setup Manager when both run at Prestage. If you have JamfConnect.pkg in Prestage, take it out and install with Setup Manager instead

**Reply by AitchySan** [reply]
interesting! ill do this now and test tomorrow. thanks!!!

**Reply by AitchySan** [reply]
Armin - just managed to get round to this. its now working. many thanks!
